services/sources-campaign.service.ts

```
@@ -0,0 +1,42 @@
+  }) => {
+    const response = await apiService.apiUrl.delete(
+      `campaigns/${req.campaignId}/sources?sourceId=${req.sourceId}`,
+      {
+        headers: {
+          "X-Organization-ID": req.organizationId,
+          Authorization: `Bearer ${req.firebaseId}`,
+        },
+      }
+    );
+  };
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)